packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -214,7 +218,7 @@ If the expression makes up the entire attribute value, you can leave off the quo
 html`<img src="/images/${this.image}">`;
 ```
 
-### Setting attributes if data is available { #attribute-available }
+### Removing an attribute under certain conditions { #removing-attribute }
 
 Sometimes you want to set an attribute only if the value or set of values is available, and otherwise remove the attribute. If the attribute value is just an empty string, use a [boolean attribute](#boolean-attribute-expressions). Sometimes, however, an attribute should be set to a specific value.
```

Comment:
Just a little bit more clarification:
```suggestion
Sometimes you want to set an attribute only if the part's value(s) meet certain conditions, and otherwise remove the attribute. For common "boolean attributes" like `disabled` and `hidden` where you want to set the attribute to an empty string for a truthy value and remove it otherwise, use a [boolean attribute](#boolean-attribute-expressions). Sometimes, however, an attribute should be set to a specific value in some cases, and removed in others.
```